Umbraco-StarterKit/Manage-users/index.md

```diff
@@ -4,4 +4,27 @@ versionFrom: 8.0.0
 
 # Manage users
 
-In this section you can find all the information and details you need in order to work with and manage both users and members on your website.
+In this section, you can find all the information and details you need in order to work with and manage both users and members on your website.
+
+## What is a User
+
+A user is someone who has access to the Umbraco backoffice and is not to be confused with Members. When Umbraco has been installed a user will automatically be generated with the login (email) and password entered during installation. Users can be created, edited, and managed in the User section.
```

Comment:
"When Umbraco has been installed a user will automatically be generated with the login (email) and password entered during installation."
Consider whether this is relevant to know, when working with Umbraco Uno.

Reply:
i feel this is relevant because we are working with people who don't necessarily know this

Reply:
Alright, but they don't actually install Umbraco themselves, which is what this sentence is referencing. Perhaps instead it should say something about how a user is created using the credentials you entered when you took a trial? Or that it's based on your existing Cloud user? 🤔